### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by alexliu724** [getlantern/lantern-forum-cn#1036] [General]
蓝灯管理员您好，
我于2024-11-03（北京时间）购买一年的会员。但是使用这几个周发现无论是电脑端还是IOS端都连接的十分吃力，要来来回回开关vpn才可以连上外网。2，连接十分不稳定，有时候可以访问有时候不行。3，因为工作原因要使用whatsapp，根本无法访问whatsapp。 所以我申请退款。附上购买记录。
![微信图片_20241114145638](https://github.com/user-attachments/assets/e28b5be1-073b-423c-88e8-7ddc8d6c8c5f)

**Comment by MaxMKMax** [comment]
@alexliu724 您好. 请您提供我账号邮箱. 谢谢

**Reply by alexliu724** [reply]
aloewater247@gmail.com

**Reply by alexliu724** [reply]
你好，请问退款有消息吗 @Mike777Khoo

**Reply by MaxMKMax** [reply]
@alexliu724 您好. 我帮您询问一下. 不好意思, 前几天我病了不在线.

**Reply by alexliu724** [reply]
@Mike777Khoo 您好，请问有消息吗。

**Comment by MaxMKMax** [comment]
@alexliu724 对了. 您用的是哪个网络服务? 我们很好奇是不是服务提供者也会影响到我们的用户.

**Reply by alexliu724** [reply]
我也不知道什么网络服务，我已经放弃这个vpn了。就帮我退款就好了，谢谢。

**Comment by MaxMKMax** [comment]
@alexliu724 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

您可以在留言区给我知道。谢谢。

**Comment by alexliu724** [comment]
@Mike777Khoo 请问怎么一点消息也没有了？？？？
